linux_os/guide/system/auditing/grub2_audit_argument/rule.yml

```diff
@@ -57,6 +57,8 @@ warnings:
         <li>On UEFI-based machines, issue the following command as <tt>root</tt>:
 {{% if product == "rhel7" %}}
         <pre>~]# grub2-mkconfig -o /boot/efi/EFI/redhat/grub.cfg</pre></li>
+{{% elif product == "ol7" %}}
```

Comment:
since `ol7` and `rhel7` are the same configuration, it would make sense to use an `or` instead of an `elif`

Reply:
@redhatrises you are right. I've decided to make it under separate condition as cfg path could be changed to ol specific. However while they are same it makes sense to keep just one instance, I'll update it.